test/e2e/installplan_e2e_test.go

```diff
@@ -107,26 +108,74 @@ var _ = Describe("Install Plan", func() {
 					counter = metric.Value
 				}
 			}
+			deprecatedCRD := &apiextensionsv1.CustomResourceDefinition{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "deprecateds.operators.io.operator-framework",
+				},
+				Spec: apiextensionsv1.CustomResourceDefinitionSpec{
+					Names: apiextensionsv1.CustomResourceDefinitionNames{
+						Plural:   "deprecateds",
+						ListKind: "DeprecatedList",
+						Singular: "deprecated",
+						Kind:     "Deprecated",
+					},
+					Scope: "Namespaced",
+					Group: "operators.io.operator-framework",
+					Versions: []apiextensionsv1.CustomResourceDefinitionVersion{
+						{
+							Name:       "v1",
+							Served:     true,
+							Storage:    true,
+							Deprecated: true,
+							Schema: &apiextensionsv1.CustomResourceValidation{
+								OpenAPIV3Schema: &apiextensionsv1.JSONSchemaProps{
+									Type: "object",
+									Properties: map[string]apiextensionsv1.JSONSchemaProps{
+										"spec": {
+											Type:        "object",
+											Description: "Spec of a test object.",
+											Properties: map[string]apiextensionsv1.JSONSchemaProps{
+												"foo": {
+													Type: "string",
+												},
+											},
+										},
+										"status": {
+											Type:        "object",
+											Description: "Spec of a test object.",
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+				Status: apiextensionsv1.CustomResourceDefinitionStatus{
+					StoredVersions: []string{"v1"},
+				},
+			}
+
+			Expect(ctx.Ctx().Client().Create(context.Background(), deprecatedCRD)).To(Succeed())
 
 			csv = newCSV(genName("test-csv-"), ns.GetName(), "", semver.Version{}, nil, nil, nil)
 			Expect(ctx.Ctx().Client().Create(context.Background(), &csv)).To(Succeed())
 
-			pdb = policyv1beta1.PodDisruptionBudget{
-				ObjectMeta: metav1.ObjectMeta{
-					Name: genName("test-pdb-"),
-				},
+			deprecated = deprecatedv1.Deprecated{
 				TypeMeta: metav1.TypeMeta{
-					Kind:       "PodDisruptionBudget",
-					APIVersion: policyv1beta1.SchemeGroupVersion.String(),
+					APIVersion: "operators.io.operator-framework/v1",
+					Kind:       "Deprecated",
+				},
+				ObjectMeta: metav1.ObjectMeta{
+					Namespace: ns.GetName(),
+					Name:      "foo",
 				},
-				Spec: policyv1beta1.PodDisruptionBudgetSpec{},
 			}
 
 			scheme := runtime.NewScheme()
-			Expect(policyv1beta1.AddToScheme(scheme)).To(Succeed())
+			Expect(deprecatedv1.AddToScheme(scheme)).To(Succeed())
 			{
 				var b bytes.Buffer
-				Expect(k8sjson.NewSerializer(k8sjson.DefaultMetaFactory, scheme, scheme, false).Encode(&pdb, &b)).To(Succeed())
+				Expect(k8sjson.NewSerializer(k8sjson.DefaultMetaFactory, scheme, scheme, false).Encode(&deprecated, &b)).To(Succeed())
 				manifest = b.String()
```

Comment:
If we can get rid of this scheme thing, we shouldn't need to import github.com/awgreene/deprecated-crd

Reply:
Done!